### PR TITLE
fixing typo (Seondary = Secondary) in design guide page

### PR DESCRIPTION
**Pull request by kkoryaka**

### CODE REVIEW

File: docs/pages/style/design.md [hunk 1]
```diff
@@ -71,26 +71,26 @@ Here’s what that looks like for recent CA.gov initiatives such as covid19.ca.g
...
 
 
-### Seondary color 
+### Secondary color 
 <div class="grid-container">
   <div class="text-center p-t-3">
     <div class="bg-secondary-color radius-round w-7 h-7 m-x-a"></div>
-    <p class="font-size-1 m-t-2 m-b-0 text-bold">Seondary color</p>
+    <p class="font-size-1 m-t-2 m-b-0 text-bold">Secondary color</p>
     <p class="font-size-1">#FF8000</p>
```
Comment (kmbrlygl) [inline docs/pages/style/design.md:79]: Caught one more hex code here (#FF8000) that should be lowercase #ff8000

Comment (kkoryaka) [reply docs/pages/style/design.md]: @kmbrlygl Fixed that one

**Comment by kmbrlygl** [comment]
Wasn't sure if you were waiting on me to merge/close this, but it looks good! Great work Konstantin!

**Review by kmbrlygl — APPROVED** [review]
Looks great! Thank you

**Pull request merged**